deploy-manage/upgrade/prepare-to-upgrade.md

```diff
@@ -22,59 +22,75 @@ To upgrade from 8.17.0 or earlier to 9.0.0, you must first upgrade to the latest
+1. To identify if your applications use unsupported features or behave differently in 9.0.0, review the deprecation logs in the Upgrade Assistant.
 
-4. Major version upgrades can include breaking changes that require additional steps to ensure your applications function as expected. Review the breaking changes for each product you use to learn more about potential impacts on your application. Ensure you test with the new version before upgrading existing deployments.
+1. Major version upgrades can include breaking changes that require additional steps to ensure your applications function as expected. Review the breaking changes for each product you use to learn more about potential impacts on your application. Ensure you test with the new version before upgrading existing deployments.
 
-5. Make the recommended changes to ensure your clients continue operating as expected after the upgrade. 
+1. Make the recommended changes to ensure your clients continue operating as expected after the upgrade.
 
     :::{note}
-       As a temporary solution, use the 8.x syntax to submit requests to 9.0.0 with REST API compatibility mode. While this allows you to submit requests using the old syntax, it doesn’t guarantee the same behavior. REST API compatibility should serve as a bridge during the upgrade, not a long-term solution. For more details on how to effectively use REST API compatibility during an upgrade, refer to [REST API compatibility](elasticsearch://reference/elasticsearch/rest-apis/compatibility.md). 
+       As a temporary solution, use the 8.x syntax to submit requests to 9.0.0 with REST API compatibility mode. While this allows you to submit requests using the old syntax, it doesn’t guarantee the same behavior. REST API compatibility should serve as a bridge during the upgrade, not a long-term solution. For more details on how to effectively use REST API compatibility during an upgrade, refer to [REST API compatibility](elasticsearch://reference/elasticsearch/rest-apis/compatibility.md).
     :::
 
-6. If you use {{es}} plugins, ensure each plugin is compatible with the {{es}} version you're upgrading.
+1. If you use {{es}} plugins, ensure each plugin is compatible with the {{es}} version you're upgrading.
+
+1. Before upgrading your production deployment, test the upgrade using an isolated environment. Ensure the test and production environments use the same settings.
+
+    :::{note}
+    The upgraded version of {{es}} may interact with its environment in different ways from the version you are currently running. It is possible that your environment behaves incorrectly in a way that does not matter to the version of {{es}} that you are currently running, but which does matter to the upgraded version. In this case, the upgraded version will not work correctly until you address the incorrect behaviour in your environment.
+    :::
+
+    :::{tip}
+    During your upgrade tests, pay particular attention to the following aspects:
```

Comment:
I see we've dispensed with the definition list approach here, very interesting. 🤓

Reply:
Yeah AIUI the recognised syntax for doing this in Markdown would be ...

```
    Term
    : Definition
```

... but apparently the variant of Markdown we use here doesn't understand that.

Reply:
it should I think: https://elastic.github.io/docs-builder/syntax/definition-lists/

Reply:
> followed by a colon and at least one space (`: `) on the next line

Experiment shows it needs to be at least three spaces.